README.md

```diff
@@ -37,8 +37,9 @@ module.exports = {
   reporters: ['dot', 'junit'],
   reporterOptions: {
     outputDir: './',
-    outputFileFormat: function(opts) { // optional
-        return `results-${opts.cid}.${opts.capabilities}.xml`
+    outputFileFormat: function(opts) {
+      var suiteName = opts.config.suite.replace(/([^a-z0-9]+)/gi, '-');
```

Comment:
How does this look @christian-bromann  ?
I'm only considering a-zA-Z0-9 as characters for the README example